README.md

```diff
@@ -21,20 +21,30 @@ Contact: [chaabane@colostate.edu](mailto:chaabane@colostate.edu). Any questions
 Accurately estimating the position of static objects, such as traffic lights, from the moving camera of a self-driving car is a challenging problem. In this work, we present a system that improves the localization of static objects by jointly-optimizing the components of the system via learning. Our system is comprised of networks that perform: 1) 5DoF object pose estimation from a single image, 2) association of objects between pairs of frames, and 3) multi-object tracking to produce the final geo-localization of the static objects within the scene. We evaluate our approach using a publicly-available data set, focusing on traffic lights due to data availability. For each component, we compare against contemporary alternatives and show significantly-improved performance. We also show that the end-to-end system performance is further improved via joint-training of the constituent models.
 
 ## Installation
-* Clone this repo, and we'll call the directory that you cloned as ${OBJECTS_GEOLOCALIZATION}
+* Clone this repo, and run the following commands.
 * Install dependencies. We use python 3.7 and pytorch >= 1.2.0
 ```
-conda create -n Objects_Geolocalization
+git clone https://github.com/MedChaabane/Static_Objects_Geolocalization.git
+cd Static_Objects_Geolocalization/
+conda create -y -n Objects_Geolocalization python=3.7
 conda activate Objects_Geolocalization
-conda install pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
-cd ${OBJECTS_GEOLOCALIZATION}
-pip install -r requirements.txt
+conda install -y pytorch==1.2.0 torchvision==0.4.0 cudatoolkit=10.0 -c pytorch
+pip install -r requirements.txt  # need to pin versions in this file

```

Comment:
That steps fail for me, could you pin all the versions in requirements.txt? @MedChaabane

Reply:
could you export the conda environment:
```
conda env export > environment.yml
```
and place that at the root of this github repo.

Reply:
yes that is done.